packages/calypso-e2e/src/lib/components/navbar-component.ts

```diff
@@ -2,7 +2,7 @@ import { Page } from 'playwright';
 
 const selectors = {
 	// Buttons on navbar
-	mySiteButton: 'text=My Site',
+	mySiteButton: 'button[data-tip-target="my-sites"]',
```

Comment:
Text becomes hidden at <480px, so the next best alternative is the data-tip-target.